test/e2e_env/kubernetes/meshcircuitbreaker/meshcircuitbreaker.go

```diff
@@ -19,36 +21,18 @@ func MeshCircuitBreaker() {
 	namespace := "meshcircuitbreaker-namespace"
 	mesh := "meshcircuitbreaker"
 
-	kubeMeshServiceYAML := fmt.Sprintf(`
-apiVersion: kuma.io/v1alpha1
-kind: MeshService
-metadata:
-  name: test-server
-  namespace: %s
-  labels:
-    kuma.io/origin: zone
-    kuma.io/mesh: %s
-    kuma.io/managed-by: k8s-controller
-    k8s.kuma.io/is-headless-service: "false"
-spec:
-  selector:
-    dataplaneTags:
-      app: test-server
-      k8s.kuma.io/namespace: %s
-  ports:
-  - port: 80
-    name: main
-    targetPort: main
-    appProtocol: http
-`, namespace, mesh, namespace)
-
 	BeforeAll(func() {
 		err := NewClusterSetup().
-			Install(MeshKubernetes(mesh)).
+			Install(
+				Yaml(
+					builders.Mesh().
+						WithName(mesh).
+						WithMeshServicesEnabled(mesh_proto.Mesh_MeshServices_Exclusive),
```

Comment:
I am not sure if that should be `Mesh_MeshServices_Exclusive` since the tests are for the `kuma.io/service` maybe `Mesh_MeshServices_Everywhere`?

Reply:
ah right I missed this one